gobblin-data-management/src/main/java/gobblin/data/management/copy/publisher/CopyDataPublisher.java

```diff
@@ -112,6 +116,8 @@ private void publishDataset(CopyableDataset copyableDataset, Collection<WorkUnit
 
     HadoopUtils.renameRecursively(fs, datasetWriterOutputPath, copyableDataset.datasetTargetRoot());
 
+    fs.delete(datasetWriterOutputPath, true);
```

Comment:
Github won't let me leave a comment at code location, but the end of this file contains a `main` method that seems to have been used for some type of test.